scripts/opt-alive.sh

```diff
@@ -10,15 +10,23 @@ set -e
 # attributor, functionattrs: inter procedural pass that deduces and/or propagates attributes
 # metarenamer: anonymizes function names
 PASSES="argpromotion deadargelim globalopt hotcoldsplit inline ipconstprop ipsccp mergefunc partial-inliner tbaa loop-extract extract-blocks safe-stack place-safepoints attributor functionattrs metarenamer lowertypetests extract-blocks openmpopt prune-eh -Os -Oz -O1 -O2 -O3"
+PASSES_SIMPLIFYLIB="-instcombine"
 
 TV="-tv"
-for p in $PASSES; do
-  for arg in $@; do
+IO_NOBUILTIN="-tv-io-nobuiltin"
```

Comment:
can we flip the default of this flag, such that if no flag is given there are no false positives. If no flag is given, io functions are marked as "unknown know function".
Only if the flag is given we mark them as regular functions. Then it's the user responsibility (or our script's) to ensure that the flag is only passed when the optimization doesn't take advantage of printf et al semantics.

Reply:
nevermind, this is what you do already. all good. sorry for the noise.